pkg/meta/lib/future_extensions.dart

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2021, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// Annotations that developers can use to extend [Future] capabilities.
+library meta_future_extensions;
+
+/// Extensions that apply to futures.
```

Comment:
That's just repeating the declaration.

Maybe:

/// Operations on futures which can help with enabling or avoiding lints.